libs/metagen/src/mdk_python/static/types.py.jinja

```diff
@@ -0,0 +1,25 @@
+import json
+from typing import List, Dict, Optional
+from dataclasses import dataclass, asdict, fields
+
+class Struct:
+    def instantiate(dt_class, val: any):
```

Comment:
Will all subclasses need to be instantiated this way?

Reply:
This will create a `dataclass` out of a raw json object so if the underlying `dataclass` has another `dataclass` field then it should recursively deserialize it into that (through `Struct.instantiate(..)`).